shell.py

```diff
@@ -144,11 +153,11 @@ def init_printing(*args, **kwargs):
 
 
 # The blueprint used to store user queries
-class Searches(db.Model):
-    user_id = db.UserProperty()
-    query = db.StringProperty(multiline=True)
-    timestamp = db.DateTimeProperty(auto_now_add=True)
-    private = db.BooleanProperty()
+class Searches(ndb.Model):
+    user_id = ndb.UserProperty()
+    Query = ndb.StringProperty()
```

Comment:
Why is the q (in Query) capitalised here?

Reply:
The ndb queries have the `query()` function. Hence `Searches.query` was ambiguous in that case. To remove the ambiguity I used `Query`.

Reply:
use `query_` instead.